opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -217,6 +218,14 @@ message DoubleHistogram {
   AggregationTemporality aggregation_temporality = 2;
 }
 
+// DoubleSummary metric data are used to convey quantile summaries,
+// a Prometheus and OpenMetrics data type. These data points cannot
```

Comment:
Should we include that these are used to also record `MinMaxSumCount` values?

Reply:
I'm not too sure, but I believe there wasn't a final decision on whether the DoubleSummary would be used to record MinMaxSumCount values. I believe it's only used for backwards compatibility with the Prometheus Summary metric for now.

Reply:
Yes, @MrAlias we decided to postpone that decision for the moment.